app/lib/adjustments.rb

```diff
@@ -0,0 +1,31 @@
+class Adjustments
+
+  attr_reader :discount_threshold, :discount_rate, :item_discount_rate, :item_on_discount
+
+  def initialize
+    @discount_threshold = 50
+    @discount_rate = 0.95
+    @item_on_discount = "Muffin"
+    @item_discount_rate = 0.90
+  end
+
+  def discount(order_subtotal)
+    order_subtotal > discount_threshold ? 
```

Comment:
Avoid multi-line ?: (the ternary operator); use `if`/`unless` instead.<br>Trailing whitespace detected.